<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSupplierDlg</class>
 <widget class="QDialog" name="AddSupplierDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
	border:0;
}
QToolButton#closeBtn{
	background-image: url(:/images/close.png);
}

QToolButton#closeBtn:hover {
	background-image: url(:/images/close_h.png);
}

QToolButton#closeBtn:pressed {	
	background-image: url(:/images/close_p.png);
}


QLineEdit ,QTextEdit{
	color:rgb(127,0,63);
	border:2px groove gray;
	border-radius:6px;
	selection-color: white;
    selection-background-color: rgb(191, 31, 127);
}

QPushButton {
    color: white;
    font: bold 10pt;
    border-image: url(:/images/button.png) 16;
    border-width: 16px;
    padding: -16px 0px;
    min-height: 32px;
    min-width: 60px;
}

QPushButton:hover{
	border-image:url(:/images/button_h.png) 16;
}
QPushButton:pressed{
	border-image:url(:/images/button_p.png) 16;
}
QLabel {
	color:gray;
	
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>供应商名(&amp;N)：</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人(&amp;P)：</string>
   </property>
   <property name="buddy">
    <cstring>contactLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话(&amp;T)：</string>
   </property>
   <property name="buddy">
    <cstring>numberLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系地址(&amp;A)：</string>
   </property>
   <property name="buddy">
    <cstring>addressLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备注(&amp;B):</string>
   </property>
   <property name="buddy">
    <cstring>noteTextEdit</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="noteTextEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定(&amp;E)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消(&amp;C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="MyImages.qrc">:/images/addSupplierHead.png</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>contactLineEdit</tabstop>
  <tabstop>numberLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>noteTextEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources>
  <include location="MyImages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddSupplierDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddSupplierDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
